contracts/orbital-core/src/admin_logic.rs

```diff
@@ -0,0 +1,51 @@
+pub(crate) mod admin {
+    use cosmwasm_std::{ensure, Addr, BlockInfo, MessageInfo, Response};
+    use cw_ownable::{assert_owner, update_ownership, Action};
+
+    use crate::{
+        contract::{ExecuteDeps, OrbitalResult},
+        error::ContractError,
+        orbital_domain::UncheckedOrbitalDomainConfig,
+        state::ORBITAL_DOMAINS,
+    };
+
+    pub fn transfer_admin(
```

Comment:
nit: wouldn't call it transfer_admin but update_ownership, as it includes also accepting the ownership, which is done by a user, not the admin, among other things